third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -39,6 +39,43 @@ resource "google_storage_bucket" "image-store" {
 }
 ```
 
+```hcl
```

Comment:
I would add a title to each of these examples explaining what the use case is. For example https://www.terraform.io/docs/providers/google/r/container_cluster.html

Reply:
Done. Added titles